### ISSUE TITLE
Bug: Unable to edit user's profile

### ISSUE THREAD

**Issue by 01dr1ch** [MISP/MISP#8002]
### Work environment

| Questions                 | Answers
|---------------------------|--------------------
| Type of issue             | Bug
| OS version (server)       | RedHat 7.9
| OS version (client)       | Win10
| PHP version               | 7.4.26
| MISP version / git hash   | 2.4.151
| Browser                   | Chrome, last patch

### Expected behavior
Editing user profile (changing any parameter) should end up with success, once GUI conditions are met.

### Actual behavior
Editing user profile, regardless if the user has ever logged in or did not yet, ends up with Internal Error (see log below).

### Steps to reproduce the behavior
- Update from 2.4.150 to 2.4.151
- runUpdates via cake as usual, diagnostics page is "green"
- Edit user's profile and click submit

### Logs, screenshots, configuration dump, ...
![image](https://user-images.githubusercontent.com/82970780/143722207-00984c81-1eb2-4e8a-8552-b8590de03d74.png)
![image](https://user-images.githubusercontent.com/82970780/143722499-59587982-8d8d-48d6-8d22-a07d223d9bb5.png)

```log
2021-11-27 22:09:17 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'User.enable_password' in 'field list'
Request URL: /admin/users/edit/20
Stack Trace:
#0 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Datasource/DboSource.php(502): PDOStatement->execute()
#1 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Datasource/DboSource.php(468): DboSource->_execute()
#2 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Datasource/DboSource.php(715): DboSource->execute()
#3 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Datasource/DboSource.php(1226): DboSource->fetchAll()
#4 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Model.php(3053): DboSource->read()
#5 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Model.php(3025): Model->_readDataSource()
#6 /var/www/MISP/app/Model/Behavior/AuditLogBehavior.php(107): Model->find()
#7 /var/www/MISP/app/Lib/cakephp/lib/Cake/Utility/ObjectCollection.php(129): AuditLogBehavior->beforeSave()
#8 /var/www/MISP/app/Lib/cakephp/lib/Cake/Event/CakeEventManager.php(244): ObjectCollection->trigger()
#9 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Model.php(1873): CakeEventManager->dispatch()
#10 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Model.php(1760): Model->_doSave()
#11 /var/www/MISP/app/Controller/UsersController.php(951): Model->save()
#12 [internal function]: UsersController->admin_edit()
#13 /var/www/MISP/app/Lib/cakephp/lib/Cake/Controller/Controller.php(499): ReflectionMethod->invokeArgs()
#14 /var/www/MISP/app/Lib/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction()
#15 /var/www/MISP/app/Lib/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke()
#16 /var/www/MISP/app/webroot/index.php(101): Dispatcher->dispatch()
#17 {main}
```

**Comment by 01dr1ch** [comment]
Hello, this still is not resolved. Behaving still the same.

**Comment by 01dr1ch** [comment]
Fixed by upgrade 2.4.152
